shadowsocks-csharp/Encryption/EncryptorFactory.cs

```diff
@@ -26,7 +26,7 @@ static EncryptorFactory()
             {
                 AEADSodiumEncryptorSupportedCiphers.Remove("aes-256-gcm");
             }
-
+#if I_KNOW_STREAM_CIPHER_IS_UNSAFE

```

Comment:
I'd rather put: if i am sb

Reply:
> 

So maybe you can tell me why we have to use aead cipher when we already have TLS by [v2ray-plugin](https://github.com/shadowsocks/v2ray-plugin). Or shadowsocks-windows is going to drop support for plugins at a later date?